### PR TITLE
GitHub pages action

### PR DESCRIPTION
**Pull request by Kingwl**
Add github pages action to build & deploy demo automatic.

### CODE REVIEW

File: scripts/build-pages.js
Comment (JYC-99) [inline scripts/build-pages.js:12]: I see there is a npm package: https://www.npmjs.com/package/gh-pages to do the copy-paste and branch management

**Comment by Kingwl** [comment]
Need further test.

File: package.json
Comment (JYC-99) [inline package.json:15]: Why can't I see the hostname for our online demo here? It suppose to be our org's github page

Comment (Kingwl) [reply package.json:15]: I'm not sure what that supposed look like. We can change that after enable the github pages.

**Comment by Kingwl** [comment]
Used `crazy-max/ghaction-github-pages` actions.

I have already tried at https://github.com/Kingwl/proposal-array-find-from-last-looking-for-stage-3-sides/blob/main/.github/workflows/pages.yml

**Comment by JYC-99** [comment]
Approved to unblock. Please manually run the action first and add link in Readme before merging.

**Review by JYC-99 — DISMISSED** [review]

**Review by JYC-99 — APPROVED** [review]

**Pull request merged**
